Fluent/Post_Process.ipynb

```diff
@@ -6,127 +6,63 @@
     "\n",
-    "Below is the recommended general structure to develop the Post_Process file for the flow of interest. Please note that we are always open to new ideas and modifying the structure below. Thus, do not limit yourself to the following structure and feel free to implement your suggestion and changes. We can modify this structure based on new feedback.\n",
+    "<img src=\"./Images/Lift_plot_combined.jpg\" width=\"900\">\n",
+    "Fig. 1. Variation of Cl with Flow Time(time step).\n",
     "\n",
-    "#### 1- Start with visualization of the residuals for most important variables within the flow field (e.g. continuity, streamwise velocity and etc.). Discuss how user should examine the residuals trends to comment on accuracy and convergence of the flow field's simulation. \n",
-    "#### 2- Explain how to visualize the contours of important normalized flow field variables (e.g. normalized velocity, pressure and etc.) on one or multiple sections of the CFD domain. Present figure(s) of visualized variable contours. \n",
-    "#### 3- Present a detail discussion on details of the presented results and link your discussion to the previously explained physics of the flow field and expected results in Physics section.\n",
-    "#### 4- Validate your simulation results via presenting a detail comparison between CFD simulation results and corresponding experimental or other numerical results. Please have in mind that the goal here is not to sell your simulation results. The goal is to highlight the limitations and capabilities of the CFD simulations and used CFD package and code.\n",
-    "#### 5- Conclude the post-processing and validation with a last conclusive paragraph highlighting the pros and cons of the CFD simulations.\n",
-    "#### 6- Cite the source(s) that you have used for writing the content of your Domain file considering the copy right rules.  Make sure to cite the sources you use or refer to within the body of the text as well as in the end!\n",
+    "In this plot alpha ($\\alpha$) is the non-dimensional rotation rate of the cylinder and is defined as $ \\alpha = R\\omega/U_\\infty$, where R is the radius of the cylinder, $\\omega$ is the rotation rate of the cylinder and $U_\\infty$ is the freestream velocity of the fluid. Figure 1 is the plot of Cl with respect to flow time (timesteps) for different $\\alpha$. It is clear that after some time, the value of Cl starts to either oscillate about a mean value or attain a constant value. This is a que to stop the iteration and know that the solution has attained a steady like solution. For $\\alpha$ = 0, it seems like the solution starts with a constant value. Don not confuse this as a steady state. There are oscillations of Cl in this range but due to the scale of the plot, they are not visible. \n",
```

Comment:
Make sure to refer to the exact numbers in the figures and diagrams. For example instead of stating "some time" state "after the 50 second the value of lift coefficient reaches 1.4 and stays almost constant". Be exact and concise!